### PR TITLE
Add source maps for debugging

### PR DESCRIPTION
**Pull request by oibe**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            "type": "node",
+            "request": "launch",
+            "program": "${workspaceRoot}/node_modules/.bin/_mocha",
+            "stopOnEntry": false,
+            // change to the name of the generated sourcemapped file in order
+            // to debug that file.
+            "args": ["-t", "10000000", "--debug-brk", "./dist/crossdock/test/tchannel_server.js"],
+            "cwd": "${workspaceRoot}",
+            "preLaunchTask": null,
+            "runtimeExecutable": "/Users/onwukikeibe/.nvm/v0.10.32/bin/node",
```
Comment (black-adder) [inline .vscode/launch.json]: not sure what this file is for, is this hardcoded line needed?

File: package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "js-yaml": "^3.6.1",
     "jsonschema": "^1.1.0",
     "request": "^2.74.0",
+    "tchannel": "^3.9.0",
```
Comment (yurishkuro) [inline package.json]: wow... not the runtime. Can it be in test deps?

Comment (oibe) [inline package.json]: we need it for the bridge...

File: .vscode/launch.json (hunk 1 above)
Comment (oibe) [inline .vscode/launch.json]: good point.  Overall this file is used as the debuging setup for vscode.

File: package.json (hunk 2 above)
Comment (oibe) [inline package.json]: we could vendor the tchannel/trace/span class

**Comment by oibe** [comment]
removed tchannel from this diff.

**Pull request merged**